src/styles/blocks/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: $card-width;
+  background-color: $color-white;
+  border: 1px solid $color-dark-white;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin-right: $card-margin-right;
+  margin-top: $card-margin-top;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(-n + 4) {
+    margin-top: 0;
+  }
+
+  &__product-image {
+    @include widthAndHeight(160px, 134px);
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    color: $color-dark;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__product-code {
+    color: $color-light;
+    align-self: flex-start;
+    font-weight: 400;
```

Comment:
it's default value, check in other places

Reply:
I have flex-direction: column on parent element, it's why I write it